<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1482043579239" stop="1482043579369">
	<name>Test Gmail</name>
	<title>Test Gmail</title>
	<test-cases>
		<test-case start="1482043579244" stop="1482043579367" status="failed">
			<name>encountered a declaration exception</name>
			<title>encountered a declaration exception</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="f151e979-5f8e-4a07-a196-ce21997fe249-attachment.png" type="image/png" size="2884"/>
			</attachments>
			<failure>
				<message>TypeError: getdata.cellFromXLS is not a function</message>
				<stack-trace>TypeError: getdata.cellFromXLS is not a function
    at Suite.&lt;anonymous&gt; (C:\Users\Dipjyoti\WebstormProjects\quickstart\e2e\PageObjects\gmailLogin.js:11:20)
    at addSpecsToSuite (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:833:25)
    at Env.describe (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:802:7)
    at describe (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3375:18)
    at Object.&lt;anonymous&gt; (C:\Users\Dipjyoti\WebstormProjects\quickstart\e2e\PageObjects\gmailLogin.js:9:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\coffee-script\lib\coffee-script\register.js:45:36)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\node_modules\jasmine\lib\jasmine.js:71:5
    at Array.forEach (native)
    at Jasmine.loadSpecs (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\node_modules\jasmine\lib\jasmine.js:70:18)
    at Jasmine.execute (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\node_modules\jasmine\lib\jasmine.js:154:8)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\frameworks\jasmine.js:129:15
    at Function.promise (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:682:9)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\frameworks\jasmine.js:101:14
    at _fulfilled (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:796:13)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:556:49
    at runSingle (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:137:13)
    at flush (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>